src/web/jdv/client/src/KMZExport.ts

```diff
@@ -37,8 +46,21 @@ async function taskPacketToKMLPlacemarks(taskPacket: LogTaskPacket) {
     const dateString = formatter.format(date)
     const bot_id = taskPacket.bot_id
 
-    const diveIconUrl = 'file://files/diveIcon.png'
-    const driftArrowHeadUrl = 'file://files/driftArrowHead.png'
+    // We omit the file:// here, so that the KMZ can be opened properly in Google Earth
+    const diveIconUrl = 'files/diveIcon.png'
+
+    
+    /**
+     * Returns the path to the appropriate embedded drift arrow icon PNG file
+     * @date 11/14/2023 - 3:14:28 PM
+     *
+     * @param {DriftPacket} drift drift packet whose speed will be used to find the appropriate icon
```

Comment:
Actually the duplication was intentional here, because the first "drift" is the name of the parameter, and the second "drift" is the first word in the description.  It's not necessary though, so I removed it.